cmd/migrations/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	"github.com/go-devs-ua/octagon/cfg"
+	"github.com/go-devs-ua/octagon/lgr"
+	_ "github.com/lib/pq"
+
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+const (
+	up   = "up"
+	down = "down"
+)
+
+func main() {
+	logger := lgr.New()
+	// defer logger.Flush()
+
+	opt, err := cfg.GetConfig()
+	if err != nil {
+		logger.Errorf("Failed to get config from .env: %+v\n", err)
+		return
+	}
+
+	db, err := connectDB(opt)
+	if err != nil {
+		logger.Errorf("%+v\n", err)
+		return
+	}
+
+	if err := migrateDB(db, logger); err != nil {
+		logger.Errorf("Failed making migrations: %+v\n", err)
+		return
+	}
+
+}
+
+// MigrateDB executes migrations.
+func migrateDB(db *sql.DB, logger *lgr.Logger) error {
+	migrations := &migrate.FileMigrationSource{
+		Dir: "./migration",
+	}
+
+	for _, arg := range os.Args {
```

Comment:
Let's try to replace up or down command receiving from `os.Args` with `flags` usage:
https://www.digitalocean.com/community/tutorials/how-to-use-the-flag-package-in-go

You should parse flags in main func and pass to `migrateDB` only it's value.

Reply:
I thought about flags, but why `os.Args` approach isn't good enough? Is it, because we can use -help with flags? Just for understanding.

Reply:
Right. It will be much more elegant way of describing possible flags when we refactor our application to be a CLI application.

On top of that, flags can be of different types - with flags libs it will be easier to use them.

Reply:
Fixed.